konduit-serving-core/src/main/java/ai/konduit/serving/pipeline/steps/ImageTransformProcessPipelineStepRunner.java

```diff
@@ -25,111 +25,97 @@
 import ai.konduit.serving.pipeline.ImageLoading;
 import ai.konduit.serving.pipeline.PipelineStep;
 import ai.konduit.serving.util.ImagePermuter;
+import javafx.util.Pair;
```

Comment:
No. Use nd4j's pair. Not sure how this got in here.

Reply:
Using it for streams. Imported nd4j's pair here.